specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -3174,6 +3178,29 @@
         }
       }
     },
+    "JupyterKernelConfig": {
+      "type": "object",
+      "description": "Jupyter kernel configuration",
+      "properties": {
+        "argv": {
+          "type": "array",
+          "description": "Argument to the the runtime",
```

Comment:
>description": "Argument to the the runtime", [](http://example.com/codeflow?start=11&length=44)

warning: passing commandline arguments through REST APIs could cause security issues - are you sanitizing these?

Reply:
not a concern given nature of service..

Reply:
This is as per kernel spec of jupyter. Hence these commandline options shouldn't be of concern https://jupyter-client.readthedocs.io/en/stable/kernels.html